### PR TITLE
[4.0] Moving comfinder from submenu to main menu

### PR DESCRIPTION
**Pull request by brianteeman**
Draft replacement for #24020 

This is just for mysql clean installs

@roland-d If this works then I will add the postgres install

Not sure I know how to do an update sql for this though due to the lft/rgt changes

### CODE REVIEW

File: administrator/components/com_finder/View/Searches/HtmlView.php [hunk 1]
```diff
@@ -102,8 +102,6 @@ public function display($tpl = null)
 			throw new GenericDataException(implode("\n", $errors), 500);
 		}
 
-		FinderHelper::addSubmenu('searches');
```
Comment (Quy) [inline administrator/components/com_finder/View/Searches/HtmlView.php:105]: Remove `use Joomla\Component\Finder\Administrator\Helper\FinderHelper;`

File: administrator/components/com_finder/finder.xml [hunk 2]
```diff
@@ -39,6 +39,22 @@
...
+		<menu img="class:finder">COM_FINDER</menu>
+		<submenu>
+			<!--
+				Note that all & must be escaped to &amp; for the file to be valid
+				XML and be parsed by the installer
+			-->
+			<menu link="option=com_finder&amp;view=index" img="class:finder"
+			      alt="Smart Search/Index">COM_FINDER_INDEX</menu>
+			<menu link="option=com_finder&amp;view=maps" img="class:finder"
+			      alt="Smart Search/Index">COM_FINDER_MAPS</menu>
```
Comment (C-Lodder) [inline administrator/components/com_finder/finder.xml]: Smart Search/Maps?

Comment (brianteeman) [reply administrator/components/com_finder/finder.xml]: Thanks @C-Lodder it was a copy paste error from the other pr

**Comment by mbabker** [comment]
> Not sure I know how to do an update sql for this though due to the lft/rgt changes

You don't, it needs to be done in PHP.  See [this method](https://github.com/joomla/joomla-cms/blob/00cd97530ed22a2c9bb94c0c3fea88cbfc056e9f/administrator/components/com_admin/script.php#L108) in 3.x script.php file.

**Comment by brianteeman** [comment]
So whoever writes the upgrade scripts can do that.

**Comment by Quy** [comment]
Revert `COM_FINDER_STATISTICS_TITLE="Smart Search Statistics"`

File: administrator/components/com_finder/View/Index/HtmlView.php [hunk 3]
```diff
@@ -21,7 +21,6 @@
 use Joomla\CMS\Router\Route;
 use Joomla\CMS\Toolbar\Toolbar;
 use Joomla\CMS\Toolbar\ToolbarHelper;
-use Joomla\Component\Finder\Administrator\Helper\FinderHelper;
```
Comment (Quy) [inline administrator/components/com_finder/View/Index/HtmlView.php]: Revert. Used on line 133.

File: administrator/components/com_finder/finder.xml [hunk 4]
```diff
@@ -39,6 +39,22 @@
...
 	</languages>
+	<administration>
+		<menu img="class:finder">COM_FINDER</menu>
+		<submenu>
+			<!--
+				Note that all & must be escaped to &amp; for the file to be valid
+				XML and be parsed by the installer
+			-->
+			<menu link="option=com_finder&amp;view=index" img="class:finder"
+			      alt="Smart Search/Index">COM_FINDER_INDEX</menu>
```
Comment (Quy) [inline administrator/components/com_finder/finder.xml:50]: Should the `alt` value be translatable?

Comment (brianteeman) [reply administrator/components/com_finder/finder.xml:50]: No. Its not alt text as you would get in an image

**Comment by brianteeman** [comment]
Closed. Not trying again.

**Pull request closed without merging**